### ISSUE TITLE
[dark mode] Error message: severe lack of colour contrast

### ISSUE THREAD

**Issue by phiw13** [philwareham/textpattern-hive-admin-theme#670]
In Dark mode, the colour contrast for “error” messages is very low.

--clr-error-text -> #ff3381
--clr-error-bkgd -> #48232b

Colour Contrast: 3.90

Does not meet WCAG standards for accessible text (min 4.5)

Desaturating `--clr-error-text` by 25% ~ 30% would improve the contrast and bring it to ~ 5.0
